purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesFactory.kt

```diff
@@ -37,7 +37,8 @@ internal class PurchasesFactory(
     fun createPurchases(
         configuration: PurchasesConfiguration,
         platformInfo: PlatformInfo,
-        proxyURL: URL?
+        proxyURL: URL?,
+        overrideBillingAbstract: BillingAbstract? = null
```

Comment:
This is a hacky approach to allow injecting a mock billing abstract for now. We might consider better approaches, but wanted to check thoughts from people before.